commit ab566816716782172ffa57c28699b50e6eea06ce
Author: Thomas Debesse <dev@illwieckz.net>
Date:   Fri Oct 21 18:22:16 2022 +0200

    patch: luxmark3/luxmark3-luxmark-fill-submission-form-from-environment.patch

diff --git a/src/submitdialog.cpp b/src/submitdialog.cpp
index 2350cd4..aec081d 100644
--- a/src/submitdialog.cpp
+++ b/src/submitdialog.cpp
@@ -65,6 +65,9 @@ SubmitDialog::SubmitDialog(
 
 	ui->setupUi(this);
 	ui->genericButton->setText("&Submit");
+	ui->nameEdit->setText(qgetenv("LUXMARK_USERNAME"));
+	ui->pwdEdit->setText(qgetenv("LUXMARK_PASSWORD"));
+	ui->noteTextEdit->setPlainText(qgetenv("LUXMARK_NOTE"));
 
 	this->setWindowTitle("LuxMark v" LUXMARK_VERSION_MAJOR "." LUXMARK_VERSION_MINOR);
 }
